src/sentry/scim/endpoints/users.py

```diff
@@ -0,0 +1,154 @@
+from django.conf import settings
+from django.db import transaction
+from django.db.models import Q
+from rest_framework.response import Response
+
+from sentry import roles
+from sentry.api.bases.organizationmember import OrganizationMemberEndpoint
+from sentry.api.endpoints.organization_member_details import OrganizationMemberDetailsEndpoint
+from sentry.api.endpoints.organization_member_index import OrganizationMemberSerializer
+from sentry.api.paginator import GenericOffsetPaginator
+from sentry.api.serializers import serialize
+from sentry.api.serializers.models.organization_member import OrganizationMemberSCIMSerializer
+from sentry.models import AuditLogEntryEvent, AuthIdentity, InviteStatus, OrganizationMember
+from sentry.signals import member_invited
+
+from .constants import SCIM_400_INVALID_FILTER, SCIM_409_USER_EXISTS
+from .utils import SCIMEndpoint, parse_filter_conditions
+
+ERR_ONLY_OWNER = "You cannot remove the only remaining owner of the organization."
+from rest_framework.exceptions import PermissionDenied
+
+from sentry.api.exceptions import ConflictError
+
+
+class OrganizationSCIMUserDetails(SCIMEndpoint, OrganizationMemberEndpoint):
+    def _delete_member(self, request, organization, member):
+        audit_data = member.get_audit_log_data()
+        if OrganizationMemberDetailsEndpoint._is_only_owner(member):
```

Comment:
Nit: If we need to call it from here, `_is_only_owner` is no longer private-by-convention. We should either delete the leading underscore or pull it up into `OrganizationEndpoint` (the common parent).

Reply:
good point. will go ahead and remove the underscore. there is a refactor to be done whereby we use the `OrganizationMemberEndpoint` for the `OrganizationMemberDetailsEndpoint` at which point we can move that method up into the parent.